resources/dataTypes.json

```diff
@@ -0,0 +1 @@
+{"sap.f.DynamicPageTitleShrinkRatio":"DynamicPageTitleShrinkRatio","sap.gantt.GenericArray":"GenericArray","sap.gantt.PaletteColor":"PaletteColor","sap.gantt.SVGLength":"SVGLength","sap.gantt.ValueSVGPaintServer":"ValueSVGPaintServer","sap.m.SelectColumnRatio":"SelectColumnRatio","sap.m.ValueCSSColor":"ValueCSSColor","sap.ui.commons.SplitterSize":"SplitterSize","sap.ui.core.AbsoluteCSSSize":"AbsoluteCSSSize","sap.ui.core.Collision":"Collision","sap.ui.core.CSSColor":"CSSColor","sap.ui.core.CSSSize":"CSSSize","sap.ui.core.CSSSizeShortHand":"CSSSizeShortHand","sap.ui.core.Dock":"Dock","sap.ui.core.ID":"ID","sap.ui.core.Percentage":"Percentage","sap.ui.core.URI":"URI","sap.ui.layout.BoxesPerRowConfig":"BoxesPerRowConfig","sap.ui.layout.cssgrid":"cssgrid","sap.ui.layout.form":"form","sap.ui.layout.GridIndent":"GridIndent","sap.ui.layout.GridSpan":"GridSpan","sap.uxap.BlockBaseColumnLayout":"BlockBaseColumnLayout","sap.ui.vtm.Matrix":"Matrix"}
```

Comment:
> `"sap.ui.layout.cssgrid":"cssgrid","sap.ui.layout.form":"form"`

Those are just namespaces, not data types, right?

Reply:
Yep! Thanks!

Just fixed it :)